Gemfile

```diff
@@ -9,17 +7,30 @@ end
 
 gem 'puma', '~> 3.7'
 gem 'rails', '~> 5.1.3'
+gem 'sidekiq'
 gem 'sqlite3'
 
 group :development, :test do
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'factory_bot_rails'
+  gem 'faker'
+  gem 'rspec-rails'
 end
 
 group :development do
+  gem 'brakeman'
   gem 'listen', '>= 3.0.5', '< 3.2'
+  gem 'rubocop-rails'
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
 
+group :test do
+  gem 'capybara'
```

Comment:
Better rspec experience is all i can say 👍

Reply:
This is part of the things i put in each project it helps with feature testing like selenium, it serves no purpose in this pull request ill be removing it